### PR TITLE
Added placement rule wise PVC selector option

### PR DESCRIPTION
**Pull request by GowthamShanmugam**
BZ: https://bugzilla.redhat.com/show_bug.cgi?id=2103979

Signed-off-by: Gowtham Shanmugasundaram <gshanmug@redhat.com>

### CODE REVIEW

**Comment by GowthamShanmugam** [comment]
image

File: locales/en/plugin__odf-console.json [hunk 1]
```diff
@@ -30,7 +30,7 @@
   "No matching application found": "No matching application found",
   "Apply DRPolicy": "Apply DRPolicy",
   "Select the applications for applying the DRPolicy": "Select the applications for applying the DRPolicy",
-  "Disaster recovery(DR) protection will be applied for all the persistent volume claims(PVCs) under the selected applications.": "Disaster recovery(DR) protection will be applied for all the persistent volume claims(PVCs) under the selected applications.",
+  "All persistent volume claims (PVCs) matching the placement rules under an application will be DR protected.": "All persistent volume claims (PVCs) matching the placement rules under an application will be DR protected.",
```
Comment (SanjalKatiyar) [inline locales/en/plugin__odf-console.json:33]:
```suggestion
  "All PersistentVolumeClaims matching the placement rules under an application will be DR protected.": "All persistent volume claims (PVCs) matching the placement rules under an application will be DR protected.",
```

Comment (GowthamShanmugam) [reply locales/en/plugin__odf-console.json:33]: ack

**Comment by GowthamShanmugam** [comment]
@SanjalKatiyar Changes done

**Pull request merged**